README.md

```diff
@@ -102,4 +102,10 @@ All source code files are licensed under the permissive zlib license
 (http://opensource.org/licenses/Zlib) unless marked differently in a particular folder/file.
 
```

Comment:
- [ ] In your README, kindly include instructions on how to run the tests: `npm test`.

Reply:
Fixed it